x-pack/plugin/ccr/src/test/java/org/elasticsearch/xpack/ccr/ShardChangesIT.java

```diff
@@ -199,70 +196,6 @@ public void testFollowIndex() throws Exception {
             assertBusy(assertExpectedDocumentRunnable(i));
         }
 
-        final int numDocsToDelete = randomIntBetween(2, 64);
-        for (int i = 0; i < numDocsToDelete; i++) {
-            client().prepareDelete("index1", "doc", Integer.toString(i)).get();
-        }
-
-        for (int i = 0; i < numDocsToDelete; i++) {
-            assertDocumentDeletedRunnable(i);
-        }
-
-        final UnfollowIndexAction.Request unfollowRequest = new UnfollowIndexAction.Request();
-        unfollowRequest.setFollowIndex("index2");
-        client().execute(UnfollowIndexAction.INSTANCE, unfollowRequest).get();
-
-        assertBusy(() -> {
-            final ClusterState clusterState = client().admin().cluster().prepareState().get().getState();
-            final PersistentTasksCustomMetaData tasks = clusterState.getMetaData().custom(PersistentTasksCustomMetaData.TYPE);
-            assertThat(tasks.tasks().size(), equalTo(0));
-        });
-    }
-
-    public void testFollowIndexWithNestedField() throws Exception {
```

Comment:
@dnhatn I do think that we want to keep this test? To make sure that we don't try to read nested Lucene documents individually?

Reply:
and just read root documents. I will make another PR for validation changes that were taken out. I think all changes in this file should remain.

Reply:
@martijnvg If you think it's needed, I am happy to put it back.

Reply:
LGTM @dnhatn!